lib/includes_for_external_validation.rb

```diff
@@ -0,0 +1,76 @@
+# gems
+require 'aasm'
+require 'devise'
```

Comment:
I know, it's less than ideal  :confused: 

This is actually necessary w.r.t. loading `Cypress::User` fixtures in our 3rd party project which is using Cypress for external validation. For the same reason, we `add_development_dependency 'devise'` in our other project's `Gemfile` and need to set all sorts of `devise`-related fields on our `Cypress::User` fixtures.

This was just a first-pass "can this even work?" approach. Definitely open to suggestions. I'd love to use a different approach if you can think of one.